tests/tmt/plans/integration-leapp.fmf

```diff
@@ -0,0 +1,16 @@
+summary: leapp JATS tests
+execute:
+    how: tmt
+discover:
+    name: jats-leapp
+    how: jats
+    url: https://gitlab.cee.redhat.com/oamg/jats-leapp
+    filter: 'tier: api,static,sanity,cli,fw,jats-leapp'
+provision:
+    how: connect
+    guest: 10.0.79.250
+    key: /home/ivasilev/.ssh/baseosci_id_rsa
```

Comment:
just curious, how this will be resolved in real world? is it possible something like "$HOME/.ssh/baseosci_id_rsa" ?

Reply:
Looks like ~ works fine, thanks!